packages/wmr/rollup.config.js

```diff
@@ -65,6 +65,14 @@ const config = {
 	// },
 	plugins: [
 		shebangPlugin(),
+		{
+			name: 'wmr-version',
+			async transform(code, id) {
+				if (/\/cli\.js$/.test(id)) {
+					return code.replace('{{VERSION}}', require('./package.json').version);
```

Comment:
I wonder if we could do this with a replace plugin https://www.npmjs.com/package/@rollup/plugin-replace

`process.env.VERSION` and replace it to that as that seems more "standard"

Reply:
Updated the PR accordingly.